### PR TITLE
Placed an open nav at the top to trigger states, while in dev

### PR DESCRIPTION
**Pull request by debughug**
# General Purpose of Pull Request:
Top nav to make it easier for us to toggle views. 

# Proposed Changes
Development QoL 

# How to test?
Click on the nav items to make sure each view triggers

image
image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
+import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom';
 import { TicTacToe } from './Routes/TicTacToe';
 import { Login } from './Routes/Login';
 import { ConnectFour } from './Routes/ConnectFour';
+import Navbar from './Layout/NavBar';
 
 export default function App() {
   return (
     <Router>
+      <Navbar></Navbar>
```
Comment (debughug) [inline src/App.js:12]: Placing NavBar at top level.

File: src/Routes/TicTacToe/index.js [hunk 2]
```diff
@@ -171,9 +170,6 @@ export class TicTacToe extends React.Component {
 
     return (
       <div className="board-wrapper">
-        <div className="nav-wrapper">
-          <NavBar />
-        </div>
```
Comment (debughug) [inline src/Routes/TicTacToe/index.js:176]: Removed navbar from tic tac toe

File: src/Layouts/NavBar/index.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```
Comment (debughug) [inline src/Layouts/NavBar/index.js:1]: The missing file :doh:

Comment (christulin) [reply src/Layouts/NavBar/index.js:1]: there we are

**Review by christulin — APPROVED** [review]

**Pull request merged**